CMP1903_A1_2324/Game.cs

```diff
@@ -4,11 +4,100 @@
 using System.Text;
 using System.Threading.Tasks;
 
+namespace CMP1903_A1_2324
```

Comment:
you have seemingly placed copies of all the other classes into the game class making it not run